javascripts/contributors.js

```diff
@@ -0,0 +1,43 @@
+(function() {
+  var words = [
+      "Oanarosca",
+      "M1guelpf",
+      "Abhi2424shek",
+      "Niccokunzmann",
+      "Rhemon",
+      "Hoangvanthien",
+      "Jig08",
+      "Hemantjadon",
+      "Gaeun",
+      "Ankitrgadiya",
+      "Princu7",
+      "Daminisatya",
+      "PolBaladas",
+      "Vickyjjj",
+      "BohJieQi",
```

Comment:
Dont use static array use the github api because in it contributors are added automatically...